kernel/src/util/random.rs

```diff
@@ -37,6 +37,14 @@ pub fn init() {
                 tail.copy_from_slice(&src[..n]);
             }
 
+            RNG.call_once(|| SpinLock::new(StdRng::from_seed(seed)));
+        } else if #[cfg(target_arch = "riscv64")] {
+            use rand::SeedableRng;
```

Comment:
SeedableRng doesn't seem to be used.

Reply:
[`StdRng::from_seed`](https://docs.rs/rand/latest/rand/rngs/struct.StdRng.html#method.from_seed) is from trait `SeedableRng`.